### ISSUE TITLE
Unable to login to wechat4u (web) via QR code

### ISSUE THREAD

**Issue by jwwang2003** [wechaty/puppet-wechat4u#38]

**Comment by jwwang2003** [comment]
Thx, Ill go check out the XP puppet provider! I used to be able to log in to web WeChat (around 2018), now it doesn't work. Could my location have something to do with this? Is it common for others to face this problem?

**Comment by huan** [comment]
> I used to be able to log in to web WeChat (around 2018), now it doesn't work. Could my location have something to do with this? Is it common for others to face this problem?

I think only Tencent can answer this question!

Good luck with trying using the XP, please let me know your result.

**Comment by jwwang2003** [comment]
> I think only Tencent can answer the above question.

Yeah, it seems like a mystery how it works in the backend. If only they can open up some official API specifically for building bots (like Discord) that would be awesome T_T.

**Comment by leochen-g** [comment]
This problem has been fixed in the next version, please use the next version. Like `npm i wechaty-puppet-wechat4u@next --save`
